libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.h

```diff
@@ -1,5 +1,10 @@
 /*
- Copyright (c) 2015-present, salesforce.com, inc. All rights reserved.
+ SFNetwork.h
```

Comment:
`SFRestAPISalesforceAction` is gone and has been replaced by `SFNetwork` instead.